service/.dockerignore

```diff
@@ -0,0 +1,6 @@
+build/
+docs/
+srv/src
+srv/target/streams
```

Comment:
I think we want to keep `srv/target/scala-2.10/` because that's the the target location `site.jar`